doc/man1/openssl-cmp.pod.in

```diff
@@ -843,7 +843,7 @@ This can be useful for batch processing and testing.
 
 =item B<-repeat> I<number>
 
-Invoke the command the given number of times with the same parameters.
+Invoke the command the given positive number of times with the same parameters.
```

Comment:
I see this doc change being a little bit silly. I cannot imagine invoking it negative number of times and 0 times would not be useful either.

Reply:
Yeah, intuitively it should be clear.
Since I added the formal range check I tried making the doc in line with  that, and it turned out to become a bit formal as well :wink:
Thanks for nevertheless approving this.